src/main.cpp

```diff
@@ -204,7 +204,7 @@ int main(int argc, char *argv[])
         // Launch the main GUI window in a separate process.
         QString exeName = QCoreApplication::applicationFilePath();
 #       ifdef _WIN32
-        if (exeName.endsWith(".com"))
+        if (exeName.toLower().endsWith(".com"))
```

Comment:
This change isn't required. It's part of an internal hack to make displaz appear to be a dual GUI/console mode program on windows. See https://github.com/c42f/displaz/blob/5e38fa70475c1b5e32caa75ada37ce4f58404d95/src/CMakeLists.txt#L146

Reply:
removed